src/Orchard.Web/Modules/Orchard.MediaLibrary.WebSearch/IBingApi.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using RestEase;
+
+namespace Orchard.MediaLibrary.WebSearch {
+    [Header("User-Agent", "RestEase")]
+    public interface IBingApi {
+        [Get("bing/v7.0/images/search")]
+        Task<string> GetImagesAsync([Header("Ocp-Apim-Subscription-Key")] string apiKey, string q);
+    }
+}
```

Comment:
Let's move this to the Api folder.

Reply:
I'm not sure which Api folder

Reply:
In a new folder in the module, but maybe Services would be better.

Reply:
Okay, It's there.